ipaserver/install/krbinstance.py

```diff
@@ -212,6 +223,10 @@ def create_instance(self, realm_name, host_name, domain_name, admin_password, ma
 
         self.__common_post_setup()
 
+        if KRB5_BUILD_VERSION >= parse_version('1.20'):
+            self.step("set PAC ticket signature support flag",
```

Comment:
Indeed but I highly doubt any distribution would build IPA with krb5 1.20+ and not set this version as a minimum version dependency. Actually, given the changes in the KDB driver ABI, running a version of IPA built with krb5 1.20+ will probably not work if installed on a system running krb5 1.19-.

Actually, my initial idea was to have a variable set with the installed krb5 version instead of the build one. But it seems it is difficult to retrieve the krb5 version in python at runtime in a portable way. The only reliable way I found would have been to use the `krb5-config` binary, which is usually provided as part of the krb5 development package (at least in the case of CentOS/RHEL/Fedora). However, I don't think we want to have krb5 development package as a runtime dependency for FreeIPA.

Reply:
krb5 1.20 adds `issue_pac()` KDB API, so freeipa package built against it will not be able to be installed on the system with krb5 1.19 or earlier simply because the dependencies will not be satisfied. We force `krb5-kdb-version` set to the one used during build. For krb5 1.20 that would be `krb5-kdb-version = 9` in the generated RPM binary package which cannot be provided by the krb5 1.19.

Reply:
For example, Fedora 37 with krb5 1.19.2:
```
# rpm -q --provides krb5-server |grep kdb
krb5-kdb-version = 8.0
# rpm -q --requires freeipa-server|grep kdb
krb5-kdb-version = 8.0
```

Reply:
Ok then perhaps we can do away with other Kerberos version checks, like https://github.com/freeipa/freeipa/blob/master/ipaserver/install/krbinstance.py#L375

Reply:
I changed the log message from `set PAC ticket signature support flag` to `enable PAC ticket signature support`.

We should keep relying on the krb5 build version, given the fact most changes on the KDB side will require a rebuild of IPA anyway.